### DISCUSSION TITLE
How to try in local network

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [kaangiray26/forte#111] [General]
Hi,
I want to try Forte in local network.
I install with docker-compose. I change only volume to my samba share. 
I  login to admin dashboard. I creat user.
I don't see any albums there.
I  try to login as user and token but failed. Do I have to set reverse proxy and ssl to login in web player?
Regards

**Comment by kaangiray26** [comment]
You don't have to configure proxies or ssl certificates to try out locally. Once you have established your own server it hosts the server instance at `0.0.0.0:3000`. Then you have to head over to https://forte.buzl.uk and enter your credentials that you have created in the dashboard along with the server address, which should be `http://localhost:3000` or just 'localhost` if you want to use the pre-registered address name in the servers branch.

I'll add a video to help with process later.

**Comment by PrzemekSkw** [comment]
Hi @kaangiray26 I try that but have login filed. I only change port for 3007.
